src/test/java/org/prebid/server/bidder/adocean/AdoceanBidderTest.java

```diff
@@ -347,6 +308,76 @@ public void makeBidsShouldReturnCorrectBidderBid() throws JsonProcessingExceptio
         assertThat(result.getValue()).doesNotContainNull().hasSize(1).element(0).isEqualTo(expected);
     }
 
+    @Test
+    public void makeBidsShouldReturnEmptyListOfBids() throws JsonProcessingException {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .id("impId")
+                        .build()))
+                .build();
+        final List<AdoceanResponseAdUnit> adoceanResponseAdUnit = asList(AdoceanResponseAdUnit.builder()
+                        .id("ad")
+                        .price("1")
+                        .winUrl("https://win-url.com")
+                        .statsUrl("https://stats-url.com")
+                        .code(" <!-- code 1 --> ")
+                        .currency("EUR")
+                        .width("300")
+                        .height("250")
+                        .crid("0af345b42983cc4bc0")
+                        .error("true")
+                        .build(),
+                AdoceanResponseAdUnit.builder()
+                        .id("adoceanmyaozpniqis")
+                        .price("1")
+                        .winUrl("https://win-url.com")
+                        .statsUrl("https://stats-url.com")
+                        .code(" <!-- code 1 --> ")
+                        .currency("EUR")
+                        .width("300")
+                        .height("250")
+                        .crid("0af345b42983cc4bc0")
+                        .error("false")
+                        .build());
+
+        final HttpCall<Void> httpCall = givenHttpCall(null, mapper.writeValueAsString(adoceanResponseAdUnit));
+
+        // when
+        final Result<List<BidderBid>> result = adoceanBidder.makeBids(httpCall, bidRequest);
+
+        // then
+        final String adm = "<script>\n"
+                + "\t\t+function() {\n"
+                + "\t\t\tvar wu = \"https://win-url.com\";\n"
+                + "\t\t\tvar su = \"https://stats-url.com\".replace(/\\[TIMESTAMP\\]/, Date.now());\n"
+                + "\n"
+                + "\t\t\tif (wu && !(navigator.sendBeacon && navigator.sendBeacon(wu))) {\n"
+                + "\t\t\t\t(new Image(1,1)).src = wu\n"
+                + "\t\t\t}\n"
+                + "\n"
+                + "\t\t\tif (su && !(navigator.sendBeacon && navigator.sendBeacon(su))) {\n"
+                + "\t\t\t\t(new Image(1,1)).src = su\n"
+                + "\t\t\t}\n"
+                + "\t\t}();\n"
+                + "\t</script> <!-- code 1 --> ";
+
+        final BidderBid expected = BidderBid.of(
+                Bid.builder()
+                        .id("ad")
+                        .impid("ao-test")
+                        .adm(adm)
+                        .price(BigDecimal.valueOf(1))
+                        .crid("0af345b42983cc4bc0")
+                        .w(300)
+                        .h(250)
+                        .build(),
```

Comment:
Redundant. 
Pls check variables usage. If they are grey in IDE they are not used